src/components/AddHosts/AddHosts.tsx

```diff
@@ -42,41 +43,43 @@ const AddHosts: React.FC = () => {
   };
 
   return (
-    <Card>
-      <CardTitle>
-        <Title headingLevel="h2" size="lg" className="card-title">
-          Host Discovery
-        </Title>
-      </CardTitle>
-      <CardBody>
-        <InventoryAddHosts />
-        <Alerts />
-        <Toolbar id="cluster-toolbar">
-          <ToolbarContent>
-            <ToolbarButton
-              variant={ButtonVariant.primary}
-              name="install"
-              onClick={handleHostsInstall}
-              isDisabled={!hasKnownHost(cluster)}
-            >
-              Install ready hosts
-            </ToolbarButton>
-            <ToolbarSecondaryGroup>
-              <EventsModalButton
-                id="cluster-events-button"
-                entityKind="cluster"
-                cluster={cluster}
-                title="Cluster Events"
-                variant={ButtonVariant.link}
-                style={{ textAlign: 'right' }}
+    <ClusterPreflightRequirementsContextProvider cluster={cluster}>
```

Comment:
Seems like `AddHosts` is missing `AlertsContextProvider` so alerts would not work

Reply:
I see `AddHosts` to be used on two places only, both are wrapped by the `AlertsContextProvider` in the hierarchy up, so we should be safe. The OCM does not reference `AddHosts` directly.